README.md

```diff
@@ -174,10 +144,10 @@ There are some example applications provided to test the server, client and boot
 The following recipes assume you are on a unix like platform and you have cmake and make installed.
 
 ### Server example
- * Create a build directory and change to that.
- * ``cmake [wakaama directory]/examples/server``
- * ``make``
- * ``./lwm2mserver [Options]``
+
+ * ``cmake -S examples/server -B build-server -DWAKAAMA_MODE_SERVER=ON``
```

Comment:
Eventually we should use the CMake presets here. But maybe it makes sense to clean up the complete CMake project first.

Reply:
Same for the other examples.